tests/test_SpiralArmsPotential.py

```diff
@@ -0,0 +1,864 @@
+from __future__ import division
+from galpy.potential import SpiralArmsPotential as spiral
+import numpy as np
+from numpy import pi
+from numpy.testing import assert_allclose
+from scipy.misc import derivative as deriv
+from astropy import units as u
+import unittest
+
+
+class TestSpiralArmsPotential(unittest.TestCase):
+
+    def test_constructor(self):
+        """Test that constructor initializes and converts units correctly."""
+        sp = spiral()  # default values
+        assert sp._amp == 1
+        assert sp._N == -2  # trick to change to left handed coordinate system
+        assert sp._alpha == -0.2
+        assert sp._r_ref == 1
+        assert sp._phi_ref == 0
+        assert sp._Rs == 0.3
+        assert sp._H == 0.125
+        assert sp._Cs == [1]
+        assert sp._omega == 0
+        assert sp._rho0 == 1 / (4 * pi)
+        assert sp.isNonAxi == True
+        assert sp.hasC == True
+        assert sp.hasC_dxdv == True
+        assert sp._ro == 8
+        assert sp._vo == 220
+
+        sp = spiral(N=3, alpha=10*u.deg, r_ref=1, phi_ref=0, Rs=0.5, H=0.5, Cs=[1], omega=0)
```

Comment:
Let's test that units are handled properly by adding a test in ``test_quantity.py`` (as mentioned in the comment to the PR). Right now the ``test_SpiralArmsPotential.py`` because astropy is not installed for that test and it's cleaner to separate the units test.